docs/docs/usage.md

```diff
@@ -7,58 +7,59 @@
 | help    | `mbt [command] --help` or<br> `mbt [command] -h`    | Prints detailed information on the selected command.                           
 <br>
 
-### How to know the tool's version
+### How to display the current tool version
 
 | Command | Usage &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;       | Description                                                    
 | ------  | --------       |  ----------                                                
-| version | `mbt -v`     | Prints the Cloud MTA Build Tool version.                                        |
+| version | `mbt -v`     | Prints the current Cloud MTA Build Tool version.                                        |
 <br>
 
 ### How to build an MTA archive from the project sources
 
 #### Prerequisites
-* `GNU Make 4.2.1` installed in your build environmnet. 
-* module build tools
+* `GNU Make 4.2.1` is installed in your build environment. 
+* Module build tools are installed in your build environment.
 
-See the correponding `Download and Installation` sections for more details.
+For more information, see the corresponding `Download` and `Installation` sections.
```

Comment:
I recommend inserting links to the  mentioned Download and Installation sections.